contrib/custom-node-e2e/README.md

```diff
@@ -0,0 +1,23 @@
+# Node e2e test with custom CRI-O builds
+
+The behaviour of [Kubernetes node e2e tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-node/e2e-node-tests.md) that use [CRI-O as the CRI](https://github.com/kubernetes/test-infra/tree/master/jobs/e2e_node/crio) which run on k8s CI could be replicated locally through GCP VM(s).
```

Comment:
let's link to a SHA instead of `master` to avoid that the link gets invalid.

Reply:
Updated, thanks!